cmd/podman/common/create.go

```diff
@@ -406,8 +406,8 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 			"Make containers root filesystem read-only",
 		)
 		createFlags.BoolVar(
-			&cf.ReadOnlyTmpFS,
-			"read-only-tmpfs", true,
+			&cf.ReadWriteTmpFS,
+			"read-write-tmpfs", true,
```

Comment:
So, I just gave this a quick look, and I'm a little confused: to my untrained eye, if a flag name changes from `read-only` to `read-write`, shouldn't its default change from `true` to `false`?